### DISCUSSION TITLE
Resource Center upload

### DISCUSSION THREAD

**Discussion by novice-gamer** [apache/dolphinscheduler#10448] [General]
Version 2.0.5
Background: Hadoop clusters are on the cloud, and the dolphin scheduler is on the local machine.
HDFS is configured in "common.properties". In the file management of the resource center, you can create folders, but creating and uploading files will get stuck.
config:
'''
[root@localhost conf]# cat common.properties |grep -Ev "^#|^$"
data.basedir.path=/tmp/dolphinscheduler
resource.storage.type=HDFS
resource.upload.path=/daas/dcfl/test/lgx
hadoop.security.authentication.startup.state=false
java.security.krb5.conf.path=/application/dolphinscheduler/conf/krb5.conf
login.user.keytab.username=hdfs-mycluster@ESZ.COM
login.user.keytab.path=/application/dolphinscheduler/conf/hdfs.headless.keytab
kerberos.expire.time=2
hdfs.root.user=hadoop
fs.defaultFS=hdfs://x.x.x.x:8020
resource.manager.httpaddress.port=8088
yarn.resourcemanager.ha.rm.ids=x.x.x.x,x.x.x.x
yarn.application.status.address=http://yarnIp1:%s/ws/v1/cluster/apps/%s
yarn.job.history.status.address=http://yarnIp1:19888/ws/v1/history/mapreduce/jobs/%s
datasource.encryption.enable=false
datasource.encryption.salt=!@#$%^&*
support.hive.oneSession=false
sudo.enable=true
development.state=false
datasource.plugin.dir=lib/plugin/datasource
'''

**Comment by jiangfeng** [comment]
the same problem， how to slove?

**Reply by hstdream** [reply]
Is there any specific error information and log?
